ipaserver/install/dsinstance.py

```diff
@@ -582,14 +582,15 @@ def __update_dse_ldif(self):
             'dse.ldif'
         )
 
-        with tempfile.NamedTemporaryFile(delete=False) as new_dse_ldif:
+        with tempfile.NamedTemporaryFile(
+                mode='w', delete=False) as new_dse_ldif:
```

Comment:
LDIFWriter never reads from its input file. There is no point opening new_dse_ldif for read and write.